pkgs/development/interpreters/pure/default.nix

```diff
@@ -40,5 +40,6 @@ stdenv.mkDerivation rec {
     platforms = with lib.platforms;
       linux;
     license = lib.licenses.gpl3Plus;
+    broken = true;
```

Comment:
please comment why it's broken.

Reply:
Pure requires LLVM 3.5 because it's based on the old JIT API that was removed in subsequent versions.